apps/SageAccountWeb/src/components/AccountSettings.tsx

```diff
@@ -16,18 +19,15 @@ import {
 import {
   SynapseClient,
   SynapseConstants,
-  Typography,
```

Comment:
Also in many places import `Typography` from MUI because the SRC version is just aliased to MUI.